### PR TITLE
Added execute immediate alias for git

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added or updated automated unit tests to verify correctness of my new code.
   * [x] I've added or updated integration tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are working by executing CLI's commands manually.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the release notes.
   * [x] I've described my changes in the section below.

### Changes description
Added execute immediate alias for git

### CODE REVIEW

**Review by sfc-gh-pczajka — DISMISSED** [review]

File: src/snowflake/cli/plugins/git/manager.py [hunk 1]
```diff
@@ -27,3 +30,21 @@ def create(
...
+
+    def execute(
+        self,
+        repository_path: str,
+        on_error: OnErrorType,
+        variables: Optional[List[str]] = None,
+    ):
+        return super().execute(
+            stage_path=repository_path, on_error=on_error, variables=variables
+        )
```
Comment (sfc-gh-turbaszek) [inline src/snowflake/cli/plugins/git/manager.py:42]: How different is this from parent's execute?

File: src/snowflake/cli/plugins/git/manager.py [hunk 2]
```diff
@@ -27,3 +30,21 @@ def create(
...
+
+    @staticmethod
+    def get_stage_name_from_path(path: str, is_git_repository: bool = False):
```
Comment (sfc-gh-turbaszek) [inline src/snowflake/cli/plugins/git/manager.py:45]: Where do we use it?

**Review by sfc-gh-turbaszek — CHANGES_REQUESTED** [review]
Got some questions around this implementation

File: src/snowflake/cli/plugins/git/manager.py (hunk 1 above)
Comment (sfc-gh-astus) [reply src/snowflake/cli/plugins/git/manager.py:42]: It isn't different :/ . Can be removed

File: src/snowflake/cli/plugins/git/manager.py (hunk 2 above)
Comment (sfc-gh-astus) [reply src/snowflake/cli/plugins/git/manager.py:45]: https://github.com/snowflakedb/snowflake-cli/pull/970/files#diff-413a7c7aa6000c534346764978e1831f392a3ee947fe572b93b228a71ee0f545L229

`is_git_repository` isn't needed

**Pull request merged**